src/main/datatypes/MQ2WindowType.cpp

```diff
@@ -860,4 +908,156 @@ bool MQ2WindowType::dataWindow(const char* szIndex, MQTypeVar& Ret)
 	return false;
 }
 
+bool MQ2InvSlotWindowType::GetMember(MQVarPtr VarPtr, const char* Member, char* Index, MQTypeVar& Dest)
+{
+	CXWnd* pWnd = static_cast<CXWnd*>(VarPtr.Ptr);
```

Comment:
You can actually use a getter here that does a lot of checks for you, like this: VarPtr.Get<CXWnd>();

Then you test that for nullptr and UI_InvSlot type

Reply:
Same as Set, strike this comment because it's more work than you need to do for this PR